packages/headless/src/features/triggers/triggers-state.ts

```diff
@@ -9,6 +9,11 @@ export interface TriggerState {
    */
   query: string;
 
+  /**
+   * The javascript function to be executed after receiving an execute trigger.
+   */
+  execute: {functionName: string; params: [string | number | boolean]};
```

Comment:
I've seen the type `[string | number | boolean]` repeated in a few places. I think it would be good to store it in a type alias and reuse, so that if we have to one day add a new type, we only need to change one place. e.g.

```
export type ExecuteTriggerParams = [string | number | boolean];
```

Reply:
I added it in `api/search/trigger.ts`